libraries/AP_HAL_ChibiOS/hwdef/common/stm32h7_mcuconf.h

```diff
@@ -294,7 +292,7 @@
 #define STM32_FDCANSEL                      STM32_FDCANSEL_PLL1_Q_CK
 #define STM32_DFSDM1SEL                     STM32_DFSDM1SEL_PCLK2
 #define STM32_SPDIFSEL                      STM32_SPDIFSEL_PLL1_Q_CK
-#define STM32_SPI45SEL                      STM32_SPI45SEL_PCLK2
+#define STM32_SPI45SEL                      STM32_SPI45SEL_PLL2_Q_CK
```

Comment:
Why these clock changes?

Reply:
I see, its for QSPI. Yeah, I am not convinced with reducing clock for QSPI Flash. We definitely need more investigation there.

Reply:
Think we are agreed we need to change now